src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Address.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Address]
+(
+	AddressId INT IDENTITY(1,1) NOT NULL PRIMARY KEY, 

```

Comment:
Can you move the definition of the primary key to a separate constraint so we can give it a name? Here's an example:

```sql
CONSTRAINT PK_Foobar PRIMARY KEY (AddressId)
```

The existing definition is absolutely fine functionally but the name of the primary key will be system generated (something like `PK__Address__091C2AFB8DEAA962`).